src/components/components/HeroSection/HeroSection.scss

```diff
@@ -0,0 +1,116 @@
+.HeroSection {
+  padding-top: 5%;
+  max-width: 1260px;
+  width: 80%;
+  max-height: 673px;
+  height: 80vh;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 3%;
+  font-family: var(--inter);
```

Comment:
do we actually need the font family here?

Reply:
no, removed it. thnx!